rxnetty/src/main/java/io/reactivex/netty/protocol/http/client/events/SafeHttpClientEventsListener.java

```diff
@@ -67,9 +67,9 @@ public void onRequestWriteFailed(long duration, TimeUnit timeUnit,
     }
 
     @Override
-    public void onResponseHeadersReceived(int responseCode) {
+    public void onResponseHeadersReceived(int responseCode, long duration, TimeUnit timeUnit) {
         if (!completed.get()) {
-            delegate.onResponseHeadersReceived(responseCode);
+            delegate.onResponseHeadersReceived(responseCode, duration, TimeUnit.MILLISECONDS);
```

Comment:
Absolutely, it should :)  looks like a copy-paste, will fix. Thanks for the review!